### DISCUSSION TITLE
How to lay out the Moon map during development

### DISCUSSION THREAD

**Discussion by kberg** [bafolts/terraforming-mars#2151] [General]
This issue was created strictly for the survey. I'm sure there are better options, but short term and long term - I'm shooting for Good Enough.

While building the Moon expansion PR #2146, I have a couple of options on where to place to Moon map.  This wouldn't necessarily be the final location of the map. Nor do I expect the map to look exactly like what comes in the expansion.

**Option 1: Below the main map. Tags move down.**

![image](https://user-images.githubusercontent.com/413481/102724930-0beba700-42e1-11eb-9be4-90509453ae9d.png)

**Option 2: To the right of Turmoil. Optionally reduce Turmoil's size**

![image](https://user-images.githubusercontent.com/413481/102724964-389fbe80-42e1-11eb-90e4-67efbd0341ee.png)

**Option 3: Tab view.**

![image](https://user-images.githubusercontent.com/413481/102724974-4c4b2500-42e1-11eb-8be4-61dc4945b6c3.png)

**Comment by chosta** [comment]
You can put it under Mars. This way it would look like Mars has its own
Luna :). Honestly moving parts around is easy so let's just start with
something and we can experiment with it as we go.

**Comment by nwai90** [comment]
I'd vote for tab view as the official game already has enough key elements to display - too much can create a visual overload and also clutter the page (imagine if there are more fan expansions with more separate boards).

One thing about these boards is that they only become relevant occasionally when you draw one of the fan expansion cards. This is different from Turmoil where you always want to follow the upcoming global events and/or jockey for party leadership.

**Comment by alrusdi** [comment]
May be tabs for all 3 components? I mean Mars, Turmoil and Luna

**Comment by ssimeonoff** [comment]
1. Don't use that map 
We should use a image with a custom grid as we do for the Mars map. 
The Standard projects will not be displayed.
The 3 params have the same scales 0-8 so they can displayed in a more compact way- e.g. only 1 scale 0-8 but a bit thicker to fit 3 tokens/numbers on a single spot. 

2. Position 
If the map looks similar to the map one with space cities around it, best fitting would be next to the main map - on the right side - main > moon > turmoil.
It is getting too much tough - maybe we should use viewport and implement 2 UI.  For mobile:  a tabular view and for desktop just display them in a flex container - the guys that play on 1440p or more, or just with a browser zoom out, will be able to see them on a single row. 

2. The sidebar 
Showing 3 extra global params might be too much. Maybe they should be displayed as x-y-z with their icons over them in a single row (too tiny icons?). 
Now I read the rules and that they are mandatory, but can be mandatory to some extend only.  No idea what to make of this.

**Comment by kberg** [comment]
This is done.
